.zuul.d/sanity.yml

```diff
@@ -0,0 +1,10 @@
+- job:
```

Comment:
The file extension is `.yml`, but it should be `.yaml`. See: https://zuul-ci.org/docs/zuul/reference/config.html#configuration-items